regression-tests.dnsdist/test_DOH.py

```diff
@@ -377,6 +377,31 @@ def testDOHQueryInvalidALPN(self):
         except:
             pass
 
+    def testDOHHTTP1(self):
+        """
+        DOH: HTTP/1.1
+        """
+        if self._dohLibrary == 'h2o':
```

Comment:
I guess `@unittest.skipIf` won't work here because `_dohLibrary` is an object property?

Reply:
I _think_ this is a reason, but I remember not being able to make it work.